soft-widget-assistant/lambda/custom/intenthandlers/LaunchRequestHandler.js

```diff
@@ -1,16 +1,48 @@
+const {
+  APP_NAME
+} = require('../utils/constants')
+
 const LaunchRequestHandler = {
   canHandle(handlerInput) {
+    console.log('xxxxxxxxx')
+    console.log(handlerInput)
```

Comment:
Hi Kulwant, can you delete these two console statements?

Reply:
Thank you !
Doing that.

Reply:
Done , can please review and merge.